week-2/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,14 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
+    return "fail";
+  }
+  if (grade >= 50) {
+    return "passed";
+  }
```

Comment:
You don't need the second `if` check here. If `grade < 50` is `false` then you already know that `grade >= 50`. If the first `if` is `true` the function will return immediately. So it could be

```
  if (grade < 50) {
    return "fail";
  }
  return "passed";
```